src/Sql/dbo/Vault/Stored Procedures/Cipher/Cipher_SoftDelete.sql

```diff
@@ -30,7 +30,9 @@ BEGIN
         [dbo].[Cipher]
     SET
         [DeletedDate] = @UtcNow,
-        [RevisionDate] = @UtcNow
+        [RevisionDate] = @UtcNow,
+        [ArchivedDate] = NULL,
+        [Archives] = NULL
```

Comment:
❓ Did we confirm that we wanted to clear all users "archived" status whenever one user soft deletes a cipher? I saw this [thread](https://bitwarden.slack.com/archives/C08AWFEADSP/p1765221800902649), but didn't see an answer